<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dev_frm</class>
 <widget class="QWidget" name="dev_frm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dev Window</string>
  </property>
  <widget class="QPushButton" name="change_led_3_btn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>assets/change-2.png</normaloff>assets/change-2.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Region 3:</string>
   </property>
  </widget>
  <widget class="QSlider" name="intensity_led_4_slider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="change_led_1_btn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>assets/change-2.png</normaloff>assets/change-2.png</iconset>
   </property>
  </widget>
  <widget class="QSlider" name="intensity_led_1_slider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Region 1:</string>
   </property>
  </widget>
  <widget class="QSlider" name="intensity_led_2_slider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Region 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Region 4:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_led_2_btn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>assets/change-2.png</normaloff>assets/change-2.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="change_led_4_btn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>assets/change-2.png</normaloff>assets/change-2.png</iconset>
   </property>
  </widget>
  <widget class="QSlider" name="intensity_led_3_slider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="on_led1_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="on_led2_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="on_led3_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="on_led4_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Target position:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="move_motor_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="target_mm_box">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
